### PR TITLE
feat(all): add cancelAllOrdersAfter

### PR DESCRIPTION
**Pull request by sc0Vu**
1. **feat(bingx): add cancelAllOrdersAfter**
```BASH
$ p bingx cancelAllOrdersAfter 100 --swap
$ p bingx cancelAllOrdersAfter 100 --swap
$ p bingx cancelAllOrdersAfter 100 --spot
$ p bingx cancelAllOrdersAfter 100 --spot

```
**NOTE:**
> * bingx required **type** parameter, which is conflict to type parameter in ccxt.
> * the interface `cancelAllOrdersAfter (timeout: number, params = {})` is not unified yet, and activated parameter not required across all exchanges 

2. **feat(bitmex): add cancelAllOrdersAfter**
```BASH
$ p bitmex cancelAllOrdersAfter 100
$ p bitmex cancelAllOrdersAfter 100
```

3. **feat(hyperliquid): add cancelAllOrdersAfter**
```BASH
$ p hyperliquid cancelAllOrdersAfter 100
$ p hyperliquid cancelAllOrdersAfter 100
```

4. **feat(woo): add cancelAllOrdersAfter**
```BASH
$ p woo cancelAllOrdersAfter 100
$ p woo cancelAllOrdersAfter 100
```

5. **feat(kraken): add cancelAllOrdersAfter**
```BASH
$ p kraken cancelAllOrdersAfter 100
$ p kraken cancelAllOrdersAfter 100
```

6. **feat(krakenfutures): add cancelAllOrdersAfter**
```BASH
$ p krakenfutures cancelAllOrdersAfter 100
$ p krakenfutures cancelAllOrdersAfter 100
```

7. **feat(okx): add cancelAllOrdersAfter**
```BASH
$ p okx cancelAllOrdersAfter 100
$ p okx cancelAllOrdersAfter 100
```

8. **feat(huobi): add cancelAllOrdersAfter**
```BASH
$ p huobi cancelAllOrdersAfter 100
$ p huobi cancelAllOrdersAfter 100
```

9. **feat(whitebit): add cancelAllOrdersAfter**
```BASH
$ p whitebit cancelAllOrdersAfter 100 '{"symbol":"BTC/USDT"}'
$ p whitebit cancelAllOrdersAfter 100 '{"symbol":"BTC/USDT"}'
```

### CODE REVIEW

File: doc/manual.rst [hunk 1]
```diff
@@ -5725,6 +5726,22 @@ Returns
...
+
+Parameters
+
+
+ * **timeout** (number) countdown time in seconds **required** on some exchanges (e.g. ``10``\ )
+ * **activated** (boolean) activated countdown **required** on some exchanges (e.g. ``true``\ )
+ * **params** (Dictionary) Extra parameters specific to the exchange API endpoint (e.g. ``{"type": "spot"}``\ )
+
+Returns
+
```
Comment (carlosmiei) [inline doc/manual.rst:5741]: @sc0Vu this is automatically generated 😬 we should add it to `manual.md`

File: ts/src/bingx.ts [hunk 2]
```diff
@@ -2753,6 +2755,48 @@ export default class bingx extends Exchange {
...
     }
 
+    async cancelAllOrdersAfter (timeout: Int, activated: Bool = undefined, params = {}) {
+        /**
+         * @method
+         * @name bingx#cancelAllOrdersAfter
+         * @description dead man's switch, cancel all orders after the given timeout
+         * @see https://bingx-api.github.io/docs/#/en-us/spot/trade-api.html#Cancel%20all%20orders%20in%20countdown
+         * @see https://bingx-api.github.io/docs/#/en-us/swapV2/trade-api.html#Cancel%20all%20orders%20in%20countdown
+         * @param {number} countdown time in seconds
```
Comment (carlosmiei) [inline ts/src/bingx.ts:2765]: @sc0Vu the default unit is milliseconds we should never accept seconds as a unified parameter

File: ts/src/bingx.ts [hunk 3]
```diff
@@ -2753,6 +2755,48 @@ export default class bingx extends Exchange {
         return response;
     }
 
+    async cancelAllOrdersAfter (timeout: Int, activated: Bool = undefined, params = {}) {
```
Comment (carlosmiei) [inline ts/src/bingx.ts:2758]: also activated is kinda ambiguous

Comment (sc0Vu) [reply ts/src/bingx.ts:2758]: I'll remove activated.

**Comment by carlosmiei** [comment]
@sc0Vu can you check the build?
image

File: ts/src/whitebit.ts [hunk 4]
```diff
@@ -1418,6 +1419,41 @@ export default class whitebit extends Exchange {
...
+        await this.loadMarkets ();
+        const symbol = this.safeString (params, 'symbol');
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancelAllOrdersAfter() requires a symbol params');
+        }
+        const market = this.market (symbol);
+        params = this.omit (params, 'symbol');
+        const request: Dict = {
+            'market': market['id'],
+            'timeout': (timeout > 0) ? this.numberToString (timeout / 1000) : null,
```
Comment (carlosmiei) [inline ts/src/whitebit.ts:1443]: we can't use null directly, but undefined will result in the same thing

**Pull request merged**